08_deploy_bmo.sh

```diff
@@ -16,6 +22,8 @@ sed -i 's/namespace: .*/namespace: openshift-machine-api/g' ocp/deploy/role_bind
 cp $SCRIPTDIR/operator_ironic.yaml ocp/deploy
 cp $SCRIPTDIR/ironic_bmo_configmap.yaml ocp/deploy
 sed -i "s#__RHCOS_IMAGE_URL__#${RHCOS_IMAGE_URL}#" ocp/deploy/ironic_bmo_configmap.yaml
+sed -i "s#provisioning_interface: \"ens3\"#provisioning_interface: \"${PRO_IF}\"#" ocp/deploy/ironic_bmo_configmap.yaml
+sed -i "s#cache_url: \"http://192.168.111.1/images\"#cache_url: \"http://${BAREMETAL_IP}/images\"#" ocp/deploy/ironic_bmo_configmap.yaml
```

Comment:
This will leave the git-tracked ironic_bmo_configmap.yaml modified, we should probably sed into a new file, or rename the one tracked in git to .yaml.template or something?

We could also then put some easier placeholders in for sed to replace?

Reply:
no, it s modifying the generated file in ocp directory